.dockerignore

```diff
@@ -1,7 +1,7 @@
-.git/*
-conf/*
-packager/*
-scripts/*
+.git
+conf
+packager
+scripts
```

Comment:
What's your docker version ?

In fact if this is version dependent, we can put them both in the `.dockerignore` as this will not break anything !